### ISSUE TITLE
NN social series

### ISSUE THREAD

**Issue by hannahkane** [MozillaFoundation/Advocacy#870]
Likely based on some awesome comments collected through the voicemail campaign

Temporarily blocked on lack of access to the VMs, but we're working on that.

@HPaulJohnson and @lovegushwa - can you provide a tentative sketch of the timeline for social?

**Comment by lovegushwa** [comment]
@hannahkane @HPaulJohnson ... social is ongoing, you will find assets here:
https://drive.google.com/drive/folders/0B2qu3ExKWIlvLXRMaFhPdWtEOHc?usp=sharing

gdoc with copy is here: 
https://docs.google.com/document/d/1yeStfQubIxDb5IH4qD9gF6dkjJf3w61QH_UgHDuayZs/edit

**Comment by hannahkane** [comment]
Initial social is done, so closing this ticket. Opened a new ticket ( #890 ) for social specifically related to the "Foxy Goes to Washington" story
